storage-blobs-python-quickstart-v4/example.py

```diff
@@ -0,0 +1,102 @@
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+
+
+
+import os, uuid, sys
+from azure.storage.blob import BlobServiceClient, BlobClient, ContainerClient, PublicAccess
+
+# ---------------------------------------------------------------------------------------------------------
+# Method that creates a test file in the 'Documents' folder.
+# This sample application creates a test file, uploads the test file to the Blob storage,
+# lists the blobs in the container, and downloads the file with a new name.
+# ---------------------------------------------------------------------------------------------------------
+# Documentation References:
+# Associated Article - https://docs.microsoft.com/en-us/azure/storage/blobs/storage-quickstart-blobs-python
+# What is a Storage Account - http://azure.microsoft.com/en-us/documentation/articles/storage-whatis-account/
+# Getting Started with Blobs-https://docs.microsoft.com/en-us/azure/storage/blobs/storage-python-how-to-use-blob-storage
+# Blob Service Concepts - http://msdn.microsoft.com/en-us/library/dd179376.aspx
+# Blob Service REST API - http://msdn.microsoft.com/en-us/library/dd135733.aspx
+# ----------------------------------------------------------------------------------------------------------
+
+
+def run_sample():
+    try:
+        # Create the BlobServiceClient that is used to call the Blob service for the storage account
+        block_blob_service = BlobServiceClient.from_connection_string(conn_str="Connection string")
+
+        # Create a container called 'quickstartblobs' and Set the permission so the blobs are public.
+        container_name ='quickstartblobs'
+        block_blob_service.create_container(container_name, public_access=PublicAccess.Container)
+
+        # Create a file in Documents to test the upload and download.
+        local_path=os.path.expanduser("~/Documents")
+        local_file_name ="QuickStart_" + str(uuid.uuid4()) + ".txt"
+        full_path_to_file =os.path.join(local_path, local_file_name)
+
+        # Write text to the file.
+        file = open(full_path_to_file,  'w')
+        file.write("Hello, World!")
+        file.close()
+
+        print("Temp file = " + full_path_to_file)
+        print("\nUploading to Blob storage as blob" + local_file_name)
+
+        # Upload the created file, use local_file_name for the blob name
+        blob_client = BlobClient.from_connection_string("Connection string", container=container_name, blob=local_file_name)
```

Comment:
since we already have a BlobServiceClient, we can just do 
`blob_client = BlobServiceClient.get_blob_client(container=container_name, blob=local_file_name)`